modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -170,8 +155,10 @@
 			}
 
 			&.e-active {
-				color: var( --n-tabs-title-color-active );
-				background-color: var( --n-tabs-title-background-color-active );
+				&:not(:hover) {
+					color: var( --n-tabs-title-color-active );
```

Comment:
I needed this to test the active state.

I have decided not to change the active tab styling on hover.